test/src/helpers/person.py

```diff
@@ -8,7 +9,7 @@ class People(BaseModel):
     members: list[schemas.Person]
 
     @classmethod
-    def alphabetical_range(cls, begin: str, end: str) -> "People":
+    def alphabetical_range(cls, begin: str, end: str,weights: Optional[list[int]] = None) -> "People":
```

Comment:
```suggestion
    def generate(cls, *, count: int, names: Optional[Iterable[str]] = None paymentWeights: Optional[Iterable[int]] = None) -> "People":
```

インターフェイスを分けたほうが拡張性高いかも？と思いました。

使用例:

```python
helpers.People.generate(count=3, names=helpers.Text.alphabetical_range("A", "C"), paymentWeight=repeat(1, 3))
```

Reply:
それはそう！！！

Reply:
これ別のヘルパー作ろうぜ的な提案だった？

Reply:
普通にそうやん日本語読めてなかった